packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -691,4 +695,62 @@ describe('TransactionService', function() {
     tr = transactionService.getCurrentTransaction()
     expect(tr.type).toBe('page-load')
   })
+
+  describe('performance entry recorder', () => {
```

Comment:
Maybe another test for the scenario if `startTransaction` is called when another non-reusable transaction already running. This would cover the scenario of making sure that `stop` is called before the `start` for the new transaction.

Reply:
Yeah sounds good, will add it.